RocketChat/Rocket.Chat
# [FIX] Actionable messages are now sent to integrations webhook

## Proposed changes (including videos or screenshots)


Actionable messages, with buttons and similar, (saved as blocks internally) were not being sent to external integrations, so CRMs and the like had no visibility on what was the actionable message sent, this PR adds a yaml version of the actions, as well as a proper json object with the same data on the webhook output, so that our clients can have more visibility in their process



## Issue(s)


## Steps to test or reproduce

 - Enable external integrations
 - Create a webhook for such integration (a simple echo server works fine)
 - Send a message with actionable items (like whatsapp templates)
 - Note that the message above is readable on the webhook

## Further comments


OC-170
